### PR TITLE
Gateway status command implementation

### PR DESCRIPTION
**Pull request by b1101**
See #209

### CODE REVIEW

File: ttnctl/cmd/gateway_status.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	Short: "Get status of a gateway",
+	Long:  `ttnctl gateway status can be used to get status of gateways.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) != 1 {
+			cmd.UsageFunc()(cmd)
+			return
+		}
+
+		euiString := args[0]
+		ctx.WithField("Gateway EUI", euiString)
```
Comment (htdvisser) [inline ttnctl/cmd/gateway_status.go]: `ctx = ctx.WithField(...`

File: ttnctl/cmd/gateway_status.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+		eui, err := types.ParseGatewayEUI(euiString)
+		if err != nil {
+			ctx.WithError(err).Fatal("Invalid Gateway EUI")
+		}
+
+		conn, manager := util.GetRouterManager(ctx)
+		defer conn.Close()
+
+		st, err := manager.GatewayStatus(util.GetContext(ctx), &router.GatewayStatusRequest{
```
Comment (htdvisser) [inline ttnctl/cmd/gateway_status.go]: Please be more explicit than `st`

File: ttnctl/cmd/gateway_status.go [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+		if t := st.Status.Time; t != 0 {
+			printKV("Reported time", time.Unix(0, t))
+		}
+		printKV("Description", st.Status.Description)
+		printKV("Platform", st.Status.Platform)
+		printKV("Contact email", st.Status.ContactEmail)
+		printKV("Region", st.Status.Region)
+		printKV("GPS coordinates", func() interface{} {
+			if gps := st.Status.Gps; gps != nil && !(gps.Latitude == 0 && gps.Longitude == 0) {
+				return fmt.Sprintf("(%v %v %v)", gps.Latitude, gps.Longitude, gps.Altitude)
```
Comment (htdvisser) [inline ttnctl/cmd/gateway_status.go]: Format as `52.373961, 4.886733` (lat, long with `%.6f`). The altitude can be left out

File: ttnctl/cmd/gateway_status.go [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+			}
+			return "not available"
+		}())
+		printKV("Rtt", func() interface{} {
+			if t := st.Status.Rtt; t != 0 {
+				return time.Duration(t)
+			}
+			return "unknown"
```
Comment (htdvisser) [inline ttnctl/cmd/gateway_status.go]: Be consistent: `not available`

File: ttnctl/cmd/gateway_status.go [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+		}())
+		printKV("Rx", fmt.Sprintf("(in: %v; ok: %v)", st.Status.RxIn, st.Status.RxOk))
+		printKV("Tx", fmt.Sprintf("(in: %v; ok: %v)", st.Status.TxIn, st.Status.TxOk))
```
Comment (htdvisser) [inline ttnctl/cmd/gateway_status.go]: Do not use `%v` formatting in these two lines, should be `%d`

Comment (b1101) [inline ttnctl/cmd/gateway_status.go]: `%v` defaults to `%d` for `uint32`, so I thought it would be acceptable.

**Pull request merged**